docs/painless/painless-literals.asciidoc

```diff
@@ -114,9 +117,9 @@ STRING: ( '"'  ( '\\"'  | '\\\\' | ~[\\"] )*? '"'  )
 "double-quoted with non-escaped 'single-quotes'"
 ----
 
-[[characters]]
+[[character-literals]]
 ==== Characters
 
-A character literal cannot be specified directly. Instead, use the
+A character literal is not specified directly. Instead, use the
 <<string-character-casting, cast operator>> to convert a `String` type value
```

Comment:
I would change this to "Character literals are not..." It just reads strange to me in the singular. :-)

Reply:
Agreed and changed.